force-app/main/default/classes/Log.cls

```diff
@@ -609,9 +631,11 @@ public without sharing class Log {
                             .summary(String.isNotBlank(componentLog.summary) ? componentLog.summary : componentLog.error != null ? componentLog.error.message : null)
                             .stackTrace(componentLog.stack)
                             .details(String.isNotBlank(componentLog.details) ? componentLog.details : componentLog.stack)
-                            .attribute(pharos__Log__c.pharos__Apex_Name__c.getDescribe().getName(), componentLog.component.name + '.' + componentLog.component.function)
-                            .attribute(pharos__Log__c.pharos__Related_Id__c.getDescribe().getName(), componentLog.recordId)
-                            .attribute(pharos__Log__c.pharos__Related_Object_Type__c.getDescribe().getName(),
+		    	    .transactionId(Log.instance.TRANSACTION_ID)
+		    	    .createIssue()
+                            .attribute(APEX_NAME_API_NAME, componentLog.component.name + '.' + componentLog.component.function)
+                            .attribute(RELATED_ID_API_NAME, componentLog.recordId)
+                            .attribute(RELATED_OBJECT_TYPE_API_NAME,
```

Comment:
@mikita-kniazeu why do we need to get object api name? This should be done with post processing